### ISSUE TITLE
Profile page - layout issues

### ISSUE THREAD

**Issue by ee61re** [meirtsvi/WazeWP7#48]
Running 0.9.8.1 on HTC Mozart.

The left edge of the Profile page is being cut off - losing first letter of each word.  Icons for Car, Privacy, Twitter and Facebook are also overlapping the text for each.

**Comment by ckeboss** [comment]
Yes, it's not been converted to silverlight UI yet. Working on it.
